extension/popup.html

```diff
@@ -12,34 +12,59 @@
       <logo-wrapper>
         <logomark></logomark>
         <logotype></logotype>
+        <button class="open-settings settings-toggle" title="Settings"></button>
       </logo-wrapper>
-      <div class="sign-up-panel hidden">
-        <main>
-          <p class="intro">Create an account or <a class="text-link sign-in-link close-popup-after-click" href="http://127.0.0.1:8000/?utm_source=addon&utm_medium=popup">sign in</a> to access your dashboard and create aliases.</p>
-          <div class="gradient-button-wrapper">
-            <a class="button create-account new-tab gradient-button close-popup-after-click" href="http://127.0.0.1:8000/?utm_source=addon&utm_medium=popup">Create Account</a>
-          </div>
-        </main>
-      </div>
-      <main class="signed-in-panel hidden">
-        <alias-list>
-          <alias-list-header>
-            <span class="alias-list-header-title">Aliases</span>
-            <span class="click-to-copy-list-header">Click to copy</span>
-          </alias-list-header>
-        </alias-list>
-        <alias-creation>
-          <div class="gradient-button-wrapper">
-            <button class="create-new-alias gradient-button" type="submit" value="Generate new relay address" data-event-label="Create New Relay Alias - No existing aliases">
-              Create New Relay Alias
-            </button>
-          </div>
-        </alias-creation>
-        <footer>
-          <p class="ff-Met aliases-remaining text-center remaining-plural">You have <span class="num-aliases-remaining"></span> remaining free <span class="alias-text">aliases</span></p>
-          <a class="button new-tab close-popup-after-click footer-button" href="http://127.0.0.1:8000/accounts/profile?utm_source=addon&utm_medium=popup"></a>
+
+      <settings>
+        <div class="settings-header">
+          <button class="close-settings settings-toggle" title="Return to main panel"></button>
+          <h1 class="settings-hl">Settings</h1>
+        </div>
+        <ul class="settings-list">
+          <li class="setting">
+            <span class="setting-label flex-col">Show Relay icon in email fields on websites</span>
+            <button title="" data-icon-visibility-option="disable-input-icon" class="toggle-icon-in-page-visibility"></button>
+          </li>
+          <li class="setting flex-col">
+            <a href="https://addons.mozilla.org/firefox/addon/private-relay/?utm_source=fx-relay-addon&utm_medium=popup&utm_campaign=beta" class="setting-label close-popup-after-click">Leave Feedback</a>
+          </li>
+        </ul>
+      </settings>
+
+      <sign-up-panel class="sign-up-panel hidden">
+        <span class="ff-Met welcome">Welcome</span>
+        <p class="intro">
+          Get started by signing in with the Firefox account you used to join the Relay waitlist.
+        </p>
+        <a class="button sign-in-btn blue-primary-btn close-popup-after-click" href="http://127.0.0.1:8000/accounts/fxa/login/?process=login">
```

Comment:
The build script should update this, but we could also fill in these href values with JS too, right? With the new `relaySiteOrigin` and/or `fxaOauthFlow` values? That would help consolidate the origin value to a single place - `background.js` - to reduce the chances of missing it.